common/parameters/Options.java

```diff
@@ -41,6 +41,7 @@
     public static final boolean DEFAULT_INFER = false;
     public static final boolean DEFAULT_TRACE_INFERENCE = false;
     public static final boolean DEFAULT_EXPLAIN = false;
+    public static final boolean DEFAULT_PERFCOUNTERS = false;
```

Comment:
`DEFAULT_REASONER_PERF_COUNTERS`
you should match the naming and also the casing (`cameCase` -> `_`)